benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -330,25 +330,22 @@ msgstr ""
 "Para funcionar correctamente, este sitio web requiere un navegador que "
 "admita JavaScript. Por favor, active JavaScript por este sitio web y"
 
-msgid "eligibility.pages.index.login_gov.label"
-msgstr "Tengo 65 años de edad o más"
-
-msgid "eligibility.pages.index.login_gov.description"
-msgstr ""
-"Usted debe de tener 65 años o más. Este beneficio no caduca, pero es posible "
-"que deba renovarlo. Usando este beneficio significa que su nueva tarifa de "
-"tránsito es la mitad de la tarifa estándar."
-
-msgid "eligibility.pages.index.mst_cc.label"
+msgid "eligibility.pages.index.courtesy_card.label"
 msgstr "Tengo una Tarjeta de Cortesía de MST"
 
-msgid "eligibility.pages.index.mst_cc.description"
+msgid "eligibility.pages.index.courtesy_card.description"
 msgstr ""
 "Esta opción es para personas que tienen una Tarjeta de Cortesía actual o una "
 "tarjeta de elegibilidad de Paratránsito de MST RIDES. Es posible que este "
```

Comment:
Same as the 1st comment - the last 2 sentences can be removed I think.

Reply:
This will be done in #1472